src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+import tensorflow as _tf
+import numpy as _np
+
+
+class ODTensorFlowModel(object):
+
+    def __init__(self, input_image_shape, batch_size, output_size, init_weights, config, is_train=True):
+        _tf.compat.v1.get_default_graph()
+
+        self.config = config
+        self.batch_size = batch_size
+        self.input_image_shape = input_image_shape
+        self.grid_shape = config['grid_shape']
+        self.box_per_cell = config['box_per_cell']
```

Comment:
Will this ever be different from the number of anchors?

Reply:
No. I forgot I was computing num_anchors as well. I will stick to using that instead and remove this.